.grunt-config/sass.js

```diff
@@ -77,6 +77,13 @@ const sass = {
 				dest: 'assets/css',
 				ext: '.css',
 			},
+			{
+				expand: true,
```

Comment:
Could/should we add a test? 

@Batsirai-muchareva @louiswol94

Reply:
In my personal opinion I don't really see a value in writing PlayWright tests for this.

Reply:
As long as shape dividers exist in some kind of test, we should be safe.
I am not sure if we have a shape divider somewhere.

It might be safe to add a template of a container and a section with the shapes.